celest/coordinates/frames/attitude.py

```diff
@@ -0,0 +1,109 @@
+
+
+from celest.coordinates.frames.base_positions import Position3d
+from celest.coordinates.ground_location import GroundLocation
+from celest.file_save import TextFileWriter
+from celest.units.core import Unit
+from celest.units.quantity import Quantity
+from celest import units as u
+import numpy as np
+
+
+class Attitude(Position3d):
+    """Satellite attitude.
```

Comment:
Add class initialization note.

```suggestion
    """Attitude(julian, roll, pitch, yaw, unit, location)
    
    Satellite attitude.
```